doc/source/user_guide/fields/field_data.rst

```diff
@@ -5,31 +5,25 @@
 Field data
 ==========
 
-You can use field data objects to access Fluent surface, scalar, vector, and
-pathlines data.
+You can use field data objects to access Fluent surface, scalar, vector, and pathlines data.
 
 Accessing field data objects
 ----------------------------
 
-In order to access field data, launch the fluent solver, and make field data
+In order to access field data, launch the Fluent solver, and make field data
```

Comment:
...ensure field data is fully available either by reading a data file, as in the following example, or by initializing the solution.

Reply:
To work with field data, ensure the Fluent solver is launched and the relevant data is made available. You can do this either by loading both case and data files or by reading a case file and initializing.